tests/foreman/longrun/test_provisioning_computeresource.py

```diff
@@ -23,88 +22,101 @@
+    rhev['rhev_img_name'] = settings.rhev.image_name
+    rhev['rhev_img_arch'] = settings.rhev.image_arch
+    rhev['rhev_img_os'] = settings.rhev.image_os
+    rhev['rhev_img_user'] = settings.rhev.image_username
+    rhev['rhev_img_pass'] = settings.rhev.image_password
+    rhev['rhev_vm_name'] = settings.rhev.vm_name
+    rhev['rhev_storage_domain'] = settings.rhev.storage_domain
+    rhev['rhv_api'] = RHEVMSystem(
+        hostname=rhev['rhev_url'].split('/')[2],
+        username=rhev['rhev_username'],
+        password=rhev['rhev_password'],
+        version='4.0',
+        verify=False,
+    )
+    rhev['cluster_id'] = rhev['rhv_api'].get_cluster(rhev['rhev_datacenter']).id
+    rhev['storage_id'] = rhev['rhv_api'].get_storage_domain(rhev['rhev_storage_domain']).id
+    rhev['network_id'] = (
+        rhev['rhv_api'].api.system_service().networks_service().list(search=f'name={bridge}')[0].id
+    )
+    if is_open('BZ:1685949'):
+        dc = rhev['rhv_api']._data_centers_service.list(search=f'name={rhev["rhev_datacenter"]}')[
+            0
+        ]
+        dc = rhev['rhv_api']._data_centers_service.data_center_service(dc.id)
+        rhev['quota'] = dc.quotas_service().list()[0].id
+    else:
+        rhev['quota'] = 'Default'
+    return rhev
+
+
+@pytest.fixture(scope="module")
+def vmware(bridge):
+    vmware = {}
+    vmware['vmware_server'] = settings.vmware.vcenter
+    vmware['vmware_password'] = settings.vmware.password
+    vmware['vmware_username'] = settings.vmware.username
+    vmware['vmware_datacenter'] = settings.vmware.datacenter
+    vmware['vmware_img_name'] = settings.vmware.image_name
+    vmware['vmware_img_arch'] = settings.vmware.image_arch
+    vmware['vmware_img_os'] = settings.vmware.image_os
+    vmware['vmware_img_user'] = settings.vmware.image_username
+    vmware['vmware_img_pass'] = settings.vmware.image_password
+    vmware['vmware_vm_name'] = settings.vmware.vm_name
+    vmware['current_interface'] = VMWARE_CONSTANTS.get('network_interfaces') % bridge
+    vmware['vmware_api'] = VMWareSystem(
+        hostname=vmware['vmware_server'],
+        username=vmware['vmware_username'],
+        password=vmware['vmware_password'],
+    )
+    vmware['vmware_net_id'] = vmware['vmware_api'].get_network(vmware['current_interface'])._moId
+    return vmware
+
+
+@pytest.fixture(scope="module")
+def provisioning(module_org, module_location):
+    os = None
+    if hasattr(settings, 'rhev') and hasattr(settings.rhev, 'image_os') and settings.rhev.image_os:
+        os = settings.rhev.image_os
+    provisioning = {}
+    provisioning['org_name'] = module_org.name
+    provisioning['loc_name'] = module_location.name
+    provisioning['config_env'] = configure_provisioning(
+        compute=True, org=module_org, loc=module_location, os=os
+    )
+    provisioning['os_name'] = provisioning['config_env']['os']
+    return provisioning
+
+
+class TestComputeResourceHost:
```

Comment:
I don't believe the test class is providing much value here, since all fixtures have been module scoped, and there don't appear to be any other test classes in this module.

Consider flattening the test module, and defining the test functions without a class.